src/main/java/lotto/model/domain/Lottos.java

```diff
@@ -27,6 +28,6 @@ public MatchCounts compareWinLotto(WinLotto winLotto) {
     }
 
     public List<Lotto> getLottos() {
-        return lottos;
+        return Collections.unmodifiableList(new ArrayList<>(lottos));
```

Comment:
새 List로 안감싸도 될 것 같아요 :)
```suggestion
        return Collections.unmodifiableList(lottos);
```